app/genesis-tool/Run.hs

```diff
@@ -144,6 +130,70 @@ runCommand
   where
     fromKMO = decideKeyMaterialOps fromVer
 
+runCommand
+  kmo
+  (DumpHardcodedGenesis
+    outDir)
+  =
+  dumpGenesis kmo outDir
+  (configGenesisData Dummy.dummyConfig)
+  $ fromMaybe (error "Hardcoded genesis lacks secrets.") (configGeneratedSecrets Dummy.dummyConfig)  
+
+runCommand
+  KeyMaterialOps{..}
+  (PrintGenesisHash
+    secretPath)
+  =
+  putStrLn =<< F.format CCr.hashHexF . unGenesisHash . snd . either (error . show) id <$> (runExceptT $ readGenesisData secretPath)
+  where
```

Comment:
Ha? `where` what?

Reply:
The line is also pretty long, could we break it so it fits into `80` characters?